packages/ember-data/lib/system/coerce-id.js

```diff
@@ -0,0 +1,9 @@
+// Used by the store to normalize IDs entering the store.  Despite the fact
```

Comment:
So, not used only by the store. Maybe something like `Internally ED normalizes id to String`?